### ISSUE TITLE
Disconnect bettween 0x Mesh in Docker container and the Browser

### ISSUE THREAD

**Issue by nuliknol** [AugurProject/augur#9168]
With latest v2 sources there is seem to be a disconnect between 0x Mesh running in the Docker container and the 0x Mesh running in the Browser. When I create an Open Order it is shown in Augur UI, but I can't get it using webserivce, webservice returns 0 orders.

Attaching screenshot for more details

![0xmesh_error](https://user-images.githubusercontent.com/1125368/91866619-54029300-ec38-11ea-885f-2362a1769e25.png)

**Comment by nuliknol** [comment]
I think this order in the Browser is being fetched from IndexDB of the browser, but not from 0x Mesh Node running locally. Because this is the error I am seeing in the console `ordersync could not get orders from enough peers (trying again` :

```
{"level":"info","msg":"core.App was started","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","time":"2020-09-01T10:37:00-05:00"}
wasm_exec.js:65 {"approxDelay_number":3600000000000,"level":"info","msg":"starting ordersync service","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","perPage_number":500,"subprotocols_array":["FilteredPaginationSubProtocol"],"time":"2020-09-01T10:37:00-05:00"}
wasm_exec.js:65 {"addresses_array":[],"level":"info","msg":"starting p2p node","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","time":"2020-09-01T10:37:00-05:00","topic_string":"/0x-orders/version/3/chain/102/schema/eyJwcm9wZXJ0aWVzIjp7Im1ha2VyQXNzZXREYXRhIjp7InBhdHRlcm4iOiIuKjZjZmMxMjVkZjdiYTI3YjI2MTM4Y2RiY2Q1ODA0MTM3ZGMzYmExYTYuKiJ9fX0="}
wasm_exec.js:65 {"bootstrapList_array":["/dns4/localhost/tcp/60558/ipfs/16Uiu2HAmRMgvPQV2UYKXuuCnNaFLpc36PhLp2UKVcL1ePseVcz4y","/dns4/localhost/tcp/60559/ws/ipfs/16Uiu2HAmRMgvPQV2UYKXuuCnNaFLpc36PhLp2UKVcL1ePseVcz4y"],"level":"info","msg":"connecting to bootstrap peers","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","time":"2020-09-01T10:37:00-05:00"}
wasm_exec.js:65 {"delayBeforeNextRetry_string":"250ms","level":"debug","minPeers_number":5,"msg":"ordersync could not get orders from enough peers (trying again soon)","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","successfullySyncedPeers_number":0,"time":"2020-09-01T10:37:00-05:00"}
wrap-log-handler.ts:8 event ZeroX:Status:Synced
wasm_exec.js:65 {"error_string":"failed to dial : all dials failed\n  * [/dns4/localhost/tcp/60559/ws] failed to negotiate security protocol: connected to wrong peer","level":"warning","msg":"failed to connect to bootstrap peer","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","peerInfo_json_string":"{\"Addrs\":[\"/dns4/localhost/tcp/60558\",\"/dns4/localhost/tcp/60559/ws\"],\"ID\":\"16Uiu2HAmRMgvPQV2UYKXuuCnNaFLpc36PhLp2UKVcL1ePseVcz4y\"}","time":"2020-09-01T10:37:00-05:00"}
wasm_exec.js:65 {"delayBeforeNextRetry_string":"500ms","level":"debug","minPeers_number":5,"msg":"ordersync could not get orders from enough peers (trying again soon)","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","successfullySyncedPeers_number":0,"time":"2020-09-01T10:37:01-05:00"}
wasm_exec.js:65 {"delayBeforeNextRetry_string":"1s","level":"debug","minPeers_number":5,"msg":"ordersync could not get orders from enough peers (trying again soon)","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","successfullySyncedPeers_number":0,"time":"2020-09-01T10:37:01-05:00"}
react-dom.development.js:332 [Violation] 'message' handler took 1361ms
ZeroXOrders.js:147 Adding orders back to mesh [{…}]
Router.js:58 CACHE HIT: getUserOpenOrders
wasm_exec.js:65 {"delayBeforeNextRetry_string":"2s","level":"debug","minPeers_number":5,"msg":"ordersync could not get orders from enough peers (trying again soon)","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","successfullySyncedPeers_number":0,"time":"2020-09-01T10:37:02-05:00"}
react-dom.development.js:332 [Violation] 'complete' handler took 465ms
react-dom.development.js:332 [Violation] 'complete' handler took 461ms
wasm_exec.js:65 {"level":"debug","msg":"added new valid order via RPC or browser callback","myPeerID":"16Uiu2HAmDN6WjjJTvSuR5HSJSQcaLMtRM34gVjwQojZDpDDAqpVT","orderHash_string":"0xab95f06b936b89333196e83d694ac0028363abc2dedceeef8eb1cd8f69e46578","time":"2020-09-01T10:37:03-05:00"}
```

**Comment by nuliknol** [comment]
I have found an easy way to reproduce this problem. Just open 2 browsers: Chrome and Firefox.
1. Create the Open Order in Chrome
2. Start Firefox and open the Market at the outcome, the order is not going to show.

The bug is still present in today's sources of v2 branch

Attaching screenshot
![mesh_bug_multibrowser_explained](https://user-images.githubusercontent.com/1125368/92981211-85f8ce00-f45e-11ea-9809-851e2f08c63a.png)

**Comment by pgebheim** [comment]
Tested, fixed in #9248

**Comment by nuliknol** [comment]
Solved
